packages/ui/src/modal/bid/TokenStatsHeader.tsx

```diff
@@ -56,12 +63,20 @@ const TokenStatsHeader: FC<Props> = ({ token, collection }) => {
           : collection?.name}
       </Text>
       {token && (
-        <Box>
+        <Box css={{ mt: '$1' }}>
```

Comment:
wonder if these should be just wrapped by a container with `mt: $1` vs applying it to each child

Reply:
Yeah that could be a nice improvement. Need to add gap as well. These changes are from Pedro's chain icon pr: https://github.com/reservoirprotocol/reservoir-kit/pull/402